app/src/common/shared/org/mozilla/vrbrowser/ui/viewmodel/WindowViewModel.java

```diff
@@ -285,6 +289,16 @@ public void onChanged(Spannable aUrl) {
         }
     };
 
+    private Observer<Spannable> mIsContentFeedObserver = new Observer<Spannable>() {
+        @Override
+        public void onChanged(Spannable o) {
+            String current = url.getValue().toString();
+            String feed = getApplication().getString(R.string.homepage_url);
```

Comment:
Can we move this to UrlUtils? We already have some helper methods there for identifying specific URLs.

Reply:
Done